edge-modules/DirectMethodReceiver/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -1,6 +0,0 @@
-ARG base_tag=2.1.2-runtime-bionic-arm32v7
```

Comment:
we should have only 1 module base image.  And since Temp Sensor is the only module included in the release; therefore I move module base image docker file under temp sensor and remove others.

Reply:
Discussed offline. This file is identical to `edge-modules/SimulatedTemperatureSensor/docker/linux/arm32v7/base/Dockerfile`, and the OneNote we use only references that one, so you deleted this one.